src/components/LoginForm.js

```diff
@@ -0,0 +1,75 @@
+import styled from "styled-components/macro";
+import icon from "../assets/icons/login_black_24dp.svg";
+import Button from "./Button";
+
+export default function LoginForm() {
+
+	function login(event) {
+		event.preventDefault();
+		const form = event.target;
+		form.reset()
+	}
```

Comment:
It is a bit unusual to start with a LoginForm. If you finish this user story, what will be the value for the user? Right now as I understand it is that the user can submit data but nothing else happens. Which is not really the point of user stories where we want to always have a finished app and add something with value for the user 🙂. 
```suggestion
export default function LoginForm({ onUserLogin }) {

	function login(event) {
		event.preventDefault();
		const form = event.target;
		const { name, password } = form.elements
		onUserLogin({name, password})
		form.reset()
	}
```

However, since you have started with this, I would suggest you implement the form correctly and maybe display the name of the user on the starting page after submitting the form. That way the form would kind of have a function for the user. Does that make sense? 🤔